content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -413,6 +414,7 @@ different Kubernetes components.
 | `PodDisruptionBudget` | `false` | Alpha | 1.3 | 1.4 |
 | `PodDisruptionBudget` | `true` | Beta | 1.5 | 1.20 |
 | `PodDisruptionBudget` | `true` | GA | 1.21 | - |
+| `PodDisruptionConditions` | `false` | Alpha | 1.25 | - |
```

Comment:
Remember to define this feature gate at the end of the file.

Reply:
Updated